calico_kubernetes/calico_kubernetes.py

```diff
@@ -300,11 +322,16 @@ def _apply_rules(self, profile_name):
         :type profile_name: string
         :return:
         """
+        try:
+            profile = self._datastore_client.get_profile(profile_name)
+        except:
+            print "Could not apply rules. Profile not found."
```

Comment:
Log out the name of the profile in this error message - it will be more useful that way.